pages/1.11/release-notes/1.11.0-rc1/index.md

```diff
@@ -17,9 +17,9 @@ To download DC/OS Enterprise, contact: [Mesosphere Support](https://support.meso
 <h3>This release candidate is for testing only and not to be used in production. </h3>
 
 
-DC/OS 1.11 Release Candidate 1 has a number of limitations that will be resolved at GA time.
+DC/OS 1.11.0 Release Candidate 1 has a number of limitations that will be resolved at GA time.
 <ul>
-<li>DC/OS 1.11 only supports new installations. Upgrades from 1.10 are not supported.</li>
+<li>DC/OS 1.11.0 only supports new installations. Upgrades from 1.10 are not supported.</li>
```

Comment:
This is probably incorrect.

Reply:
Yup, this line needs to be removed. 
Upgrade is supported from 1.10.5 version of DC/OS.

Reply:
@pavisandhu  Let's remove this line.

```
-<li>DC/OS 1.11 only supports new installations. Upgrades from 1.10 are not supported.</li>
+<li>DC/OS 1.11.0 only supports new installations. Upgrades from 1.10 are not supported.</li>
```

Also, I think, we should remove the release notes for RC in the docs publishing itself.
We should only publish for 1.11.0 and should avoid confusing customers with exposing them to RC1 and RC4 release notes. WDYT?